packages/mux-player/src/index.ts

```diff
@@ -1145,6 +1145,19 @@ class MuxPlayerElement extends VideoApiElement {
     this.media.metadata = { ...this.metadataFromAttrs, ...val };
   }
 
+  async addCuePoints<T = any>(cuePoints: { timestamp: number; value: T }[]) {
```

Comment:
thought: should passing in `null` remove all cuepoints? Alternatively, have a separate `removeCuePoints` method that you can pass in the cuepoints to remove. So, like `removeCuePoints(cuePoints())` would remove all.